README.md

```diff
@@ -110,6 +110,9 @@ The following table lists the configurable parameters of the nifi chart and the
 | `sts.AntiAffinity`                                                          | Affinity for pod assignment                                                                                        | `soft`                          |
 | `sts.pod.annotations`                                                       | Pod template annotations                                                                                           | `security.alpha.kubernetes.io/sysctls: net.ipv4.ip_local_port_range=10000 65000`                          |
 | `sts.hostAliases    `                                                       | Add entries to Pod /etc/hosts                                                                                      | `[]`                            |
+| `sts.startupProbe.enabled`                                                  | enable Startup Probe on Nifi server container                                                                      | `false`                            |
```

Comment:
Would you be ok to add a reference to the Kubernetes doc as a link on "Startup Probe"?

https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#define-startup-probes

Reply:
@banzo good idea.  yes, i will add the link.

Reply:
@banzo readme has been updated